release-notes/8.0/preview/api-diff/rc1/Microsoft.NETCore.App/8.0-rc1_System.Numerics.md

```diff
@@ -0,0 +1,15 @@
+# System.Numerics
+
+``` diff
+ namespace System.Numerics {
+     public static class Vector {
+-        public static Vector<T> LoadUnsafe<T>(ref T source);
++        public static Vector<T> LoadUnsafe<T>([RequiresLocationAttribute] ref T source);
```

Comment:
Can we search/replace all of these `[RequiresLocationAttribute] ref` with `ref readonly`?